packages/tokamak/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -355,8 +403,7 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
         // Set the l2Sender so contracts know who triggered this withdrawal on L2.
         l2Sender = _tx.sender;
         if (_tx.value > 0) {
-            IERC20(_nativeTokenAddress).safeIncreaseAllowance(_tx.target, _tx.value);
-            depositedAmount -= _tx.value;
+            IERC20(_nativeTokenAddress).safeIn(_tx.target, _tx.value);
```

Comment:
I think we can fix it! 🙇🏻‍♂️🙇🏻‍♂️

Reply:
Thank you very much, @rlgns98kr 
I updated!